src/persist/src/indexed/arrangement.rs

```diff
@@ -973,24 +973,30 @@ mod tests {
     }
 
     // Keys are randomly generated, so clear them before we do any comparisons.
-    fn cleared_unsealed_keys(batches: &[UnsealedBatchMeta]) -> Vec<UnsealedBatchMeta> {
+    // Also clear size_bytes so the tests don't have to be updated every time
```

Comment:
love this

Reply:
Yeah, I saw this one coming :)

https://github.com/MaterializeInc/materialize/blob/8cdf118ddd8de7825c4f665cf43f7c8ce611f3b3/src/persist/src/indexed/arrangement.rs#L1356